betty/concurrent.py

```diff
@@ -16,6 +17,13 @@
 
 from betty.asyncio import gather
 
+# @todo What is a good production value?
+# @todo The problem is perhaps (but maybe it is) not so much how many files we open in our own code
+# @todo but DEFINITELY the number of multiprocessing locks we use, each of which uses a pipe, which is a file handle
+# @todo
+# @todo Do we still need this?
+MAX_OPEN_FILES = multiprocessing.Manager().BoundedSemaphore(512)
```

Comment:
We port this to processes in the generation pool, but not other processes. What about other threads?

Reply:
Remove this. It's useless because of the differences between operating systems